### PR TITLE
IBX-386: Extended DAM configuration

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-386
| Versions      | 3.2+

### CODE REVIEW

**Review by mateuszbieniek — CHANGES_REQUESTED** [review]

**Review by mateuszbieniek — APPROVED** [review]

File: docs/guide/config_connector.md [hunk 1]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+    {% set transformation = ibexa_platform_dam_image_transformation(dam_image.source, '770px') %}
+    {% set asset = ibexa_platform_asset(dam_image.destinationContentId, dam_image.source, transformation) %}
+    {% set image_uri = asset.assetUri.path %}
+    <img src="{{ image_uri }}">
+{% endif %}
+```
+
+The `770px` parameter in the template above is used to render the DAM image. It is the `unsplash` specific image variation and must be defined separately.
+
+Next, define the created custom template for embed viewType matched for the DAM Image Content Type. For more information about displaying content, see [Content rendering](../guide/content_rendering.md).
```
Comment (dabrt) [inline docs/guide/config_connector.md:41]:
```suggestion
Next, in `config/packages/ezplatform.yaml`, set the `dam.html.twig` template for the `embed` view type that is matched for the Content Type, which you created for DAM images. 
For more information about displaying content, see [Content rendering](../guide/content_rendering.md).
```
"for the Content Type that you create for DAM images" - unless DXP comes with a built-in content type for DAM images?

File: docs/guide/config_connector.md [hunk 2]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+     site:
+       content_view:
+         embed:
+           image_dam:
+             template: embed/dam.html.twig
+             match:
+               Identifier\ContentType: <dam_image_content_type_identifier>
+```
+
+In the `config/packages/ezplatform.yaml` add the following configuration:
```
Comment (dabrt) [inline docs/guide/config_connector.md:55]:
```suggestion
In the same YAML file, add the following configuration:
```

File: docs/guide/config_connector.md [hunk 3]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+    utm_source: <your_utm_source_name>  
+    variations:
+       770px:
+            fm: jpg
+            q: 80
+            w: 770
+            fit: max
+```
+
+You can customize the parameters according to your needs. For more information about supported parameters, see the [Unsplash documentation](https://unsplash.com/documentation#dynamically-resizable-images).
```
Comment (dabrt) [inline docs/guide/config_connector.md:69]:
```suggestion
You can customize the parameters according to your needs. 
For more information about supported parameters, see the [Unsplash documentation](https://unsplash.com/documentation#dynamically-resizable-images).
```

File: docs/guide/config_connector.md [hunk 4]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+
+In the Back Office go to **Admin** > **Content Types** and in the **Content** group create a Content Type with the Image Asset Field.
```
Comment (dabrt) [inline docs/guide/config_connector.md:71]:
```suggestion
In the Back Office, go to **Admin** > **Content Types**.
In the **Content** group, create a Content Type for DAM images, which includes the ImageAsset Field. 
```
Image Asset or ImageAsset?

Shouldn't the content type be created beforehands if we are supposed to provide the <dam_image_content_type_identifier> above?

File: docs/guide/config_connector.md [hunk 5]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+            fit: max
+```
+
+You can customize the parameters according to your needs. For more information about supported parameters, see the [Unsplash documentation](https://unsplash.com/documentation#dynamically-resizable-images).
+
+In the Back Office go to **Admin** > **Content Types** and in the **Content** group create a Content Type with the Image Asset Field. 
+
+Now, when you use the Embed block in the Page Builder, you should see a DAM Image.
+
+For more information about blocks customization (defined templates, variations), see [Create custom block](../tutorials/enterprise_beginner/4_create_a_custom_block.md).
```
Comment (dabrt) [inline docs/guide/config_connector.md:75]:
```suggestion
For more information about block customization (defined templates, variations), see [Create custom block](../tutorials/enterprise_beginner/4_create_a_custom_block.md).
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/guide/config_connector.md [hunk 6]
```diff
@@ -17,3 +17,59 @@ The configuration for each connector depends on the requirements of the specific
...
+{% set dam_image = ez_field_value(content, 'image') %}
+{% if dam_image.source is not null %}
+    {% set transformation = ibexa_platform_dam_image_transformation(dam_image.source, '770px') %}
+    {% set asset = ibexa_platform_asset(dam_image.destinationContentId, dam_image.source, transformation) %}
+    {% set image_uri = asset.assetUri.path %}
+    <img src="{{ image_uri }}">
+{% endif %}
+```
+
+The `770px` parameter in the template above is used to render the DAM image. It is the `unsplash` specific image variation and must be defined separately.
```
[the remark this answers is not in the document]
Comment (juskora) [reply docs/guide/config_connector.md:39]: In that case, the term should stay as it is -> parameter, not value. I'm aware of this term but here it should be also used as the image variation but with the note that it is specific for the unsplash.

**Review by dabrt — APPROVED** [review]

**Pull request merged**